README.md

```diff
@@ -0,0 +1,130 @@
+[![Build Status][build-shield]][build-url]
```

Comment:
I think we don't have a CI to fetch build status

Reply:
I'll remove it in the next push